prog/vm/host_nexus.rb

```diff
@@ -1,18 +1,28 @@
 # frozen_string_literal: true
 
 class Prog::Vm::HostNexus < Prog::Base
-  subject_is :sshable, :vm_host
+  subject_is :vm_host
```

Comment:
That's odd...is the subject no longer sshable? Did this change sneak in from conflict resolution?

Reply:
yes, it sneaked in